.github/ISSUE_TEMPLATE/data_access_request.yml

```diff
@@ -0,0 +1,44 @@
+name: Data access request
+description: Request access to pseudopeople input data (beyond the sample included with pseudopeople)
+body:
+  - type: input
+    attributes:
+      label: "What is the name of your project?"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "What is the purpose of your project?"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "Who is involved in the project? Which of these people will have direct access to the pseudopeople input data?"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "What funding is the project under? What expectations with respect to open access and access to data come with that funding?"
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: "We commit to:"
+      options:
+        - label: be responsive to further questions from interested parties
+          required: true
+        - label: deprecate and replace our version of the pseudopeople input data when a new version is released
```

Comment:
Is this really something we're "requiring" of the users?

Reply:
Yes, it does seem like a very stringent requirement, but this is what we wrote for the data access process. It would be nice to get context from @Ironholds about this.

Reply:
What's the impetus for this commitment? Is it to avoid needing to be backward compatible with the input data?

Reply:
It's more about making sure lines of communication are open between us and the data users, I think.